usmqe_tests/api/gluster/conftest.py

```diff
@@ -3,22 +3,13 @@
 from usmqe.gluster import gluster
 import usmqe.inventory as inventory
 
-
-@pytest.fixture
-def valid_cluster_id(valid_session_credentials):
-    """
-    Generate valid id of imported cluster.
-    """
-    # TODO change
-    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
-    cluster_list = api.get_cluster_list()
-    return cluster_list[0]["cluster_id"]
```

Comment:
`valid_cluster_id` fixture is removed at the same time `invalid_cluster_id` is not. And both are used in tests even the `valid` one.